stan/math/prim/meta/max_size_mvt.hpp

```diff
@@ -1,34 +1,35 @@
 #ifndef STAN_MATH_PRIM_META_MAX_SIZE_MVT_HPP
 #define STAN_MATH_PRIM_META_MAX_SIZE_MVT_HPP
 
-#include <stan/math/prim/meta/length_mvt.hpp>
+#include <stan/math/prim/meta/size_mvt.hpp>
 #include <cstdlib>
 
 namespace stan {
+namespace math {
```

Comment:
Same as above, and note that `size_mvt` is inside `namespace stan` only.

Reply:
See answer above.